src/blockchain.js

```diff
@@ -59,19 +66,29 @@ class Transaction{
 }
 
 class Block {
+    /**
+     * @param {Date} timestamp 
```

Comment:
This is not a `Date` object. We pass here the timestamp of dates (which is a number).

Reply:
resolved in the conversation below